cli/src/lib.rs

```diff
@@ -306,7 +309,7 @@ impl Iroha {
         if Self::start_telemetry(telemetry, &config).await? {
             iroha_logger::info!("Telemetry started")
         } else {
-            iroha_logger::error!("Telemetry did not start")
+            iroha_logger::warn!("Telemetry not started")
```

Comment:
`info` maybe?

Reply:
The logic is as follows: 
- Use `trace` for showing information about normal operation, specifically which functions were called and returned from. 
- Use `debug` for information that you'd `println` if you were debugging a misbehaviour. Diagnostics, information status updates. E.g. if debugging deadlocks, calling `debug!("lock X locked")`, is a sane option. Don't overdo it, because `debug` statements add a conditional operation sometimes the branch predictor doesn't optimise them away. 
- Use `info` for regular feedback on normal processes. We have too much there currently. This information is supposed to tell node administrators what's going on, but not much else. It's only useful during startup and shutdown, because everything else falls either into debugging or error reporting. 
- Use `warn` for things that *could* be errors and node administrators should investigate, but which can also be considered part of the normal operation. Example: you could have not requested telemetry be initialised. Or you did request it, and it couldn't be initialised. The latter point should not stop Iroha from working, but is worth investigating. 
- Use `error` for things that require manual intervention from the node administrator. For example, disk is nearing full. Communication with an actor is supposed to occur, but it didn't. 

If we follow this, you can attach an alert to finding a `warn` or higher message. What's more important, to improve perofmrnace we set the logging to `error` in real deployments. So if you need the telemetry having the message in `warn` could tell you that it didn't start, because e.g. you forgot to enable it. Usually, you only need to disable telemetry when you need performance, like `unsafe` in Rust, so producing a warning makes sense.

Reply:
Yeah, i agree, thank you for clarification!

Reply:
since you went through the trouble of explaining this in great detail, it would be a good idea to add this section to the README

Reply:
CONTRIBUTING probably